### DISCUSSION TITLE
What to show all the prediction of all the models being trained?

### DISCUSSION THREAD

**Discussion by skfjskfj** [winedarksea/AutoTS#250] [General]
In order to compare each model, how to get the prediction and plot all the models being trained to compare with each model?

**Comment by winedarksea** [comment]
You really want to sit down and manually compare 20,000 different models? That's about the normal number I run in 12 hours or so for a lot of scripts.

You can look at the best model like this:
```python
ax = model.plot_validations(use_df, subset='Best')
plt.legend(bbox_to_anchor=(1.02, 1), loc='upper left', borderaxespad=0)
# plt.savefig("validation_plot2.png", dpi=300, bbox_inches="tight")
plt.show()

ax = model.plot_validations(use_df, subset='Worst')
plt.legend(bbox_to_anchor=(1.02, 1), loc='upper left', borderaxespad=0)
plt.show()
```
or view `model.results()` or `model.results("validation")` to see a summary table of results by model tried.

if you really do want to view the plot of a given model in that table, you can use the `models` arg of `plot_validations` and put the `ID` from that table in there to view the prediction for it.
